Documentation/FontEditor/UsingFontEditor.md

```diff
@@ -0,0 +1,123 @@
+
+- The ***right section*** which displays both search input for unicode blocks and the unicode block selection list.
+Metadata and unicode blocks can be turned on or off via ` Menu -> View ` option. Unicode block filters only what to display contained within the range of the block. Basic Latin covers 000-007F, Latin-1 Supplement covers 0080-00FF, Latin Extended-A covers 0100-017F and so on and so forth. Global search for glyph is affected when a certain block is currently selected. Only by selecting ` Show All ` will the global glyph search work as expected. So make it a habit of confirming that Show All is active before searching for a glyph.
+
+### The status bar
+The status bar displays additional information describing what each menu entry and toolbar icon does. It identifies what is the current unicode value of the glyph the cursor is in. It shows the glyph's visual representation (if available), glyph description and glyph dimensions. Second only to the glyph editor window, the status bar is your next best friend on your path to becoming a font master.
+
+### Create New Font
+![](FontEditor_New_Font.png)
+
+To create a new font, you can either click on New Font icon on the Toolbar or go to ` File -> New Font ` on the Menubar.
+A wizard will walk you into preparing the needed parameters for your new font.
+
+### Typeface Properties
+![](FontEditor_Typeface_properties.png)
+
+You can try out the default values just to get the feel of the program.
+
+### Glyph Properties
+![](FontEditor_Glyph_properties.png)
+
+Just click **Finish** if you are so inclined.
+
+### Edit Glyph Properties
+![](FontEditor_Edit_Glyph_properties.png)
+
+Adjust values to suit your needs. The higher the value, the larger the font size.
+
+### Untitled font
+![](FontEditor_Untitled.png)
+
+Congratulations on your successful initial font setup. Now you are ready to begin. And begin you shall. There is more to font creation than simply scribbling away. You need to always remind yourself this question: "Where is the fun in that?". Start with what you need to do while striving to achieve fun in the process. Let's be honest, font creation is one of the many thankless jobs, unless and until there is fun in it, why bother?
+
+### Launch another instance
+![](FontEditor_Launching_second_instance.png)
+
+Figure above shows where you can find FontEditor from inside the SerenityOS desktop. It is joined together with other similarly amazing applications like GML Playground, Hack Studio, Hex Editor, Inspector, Keyboard Mapper, Profiler and Theme Editor. Each of the apps by themselves are worthy of their own self-contained tutorial which we hope to soon see materialize. These apps are all native to SerenityOS as they are for SerenityOS, by SerenityOS and of SerenityOS. Python was included simply because this is the `Development` section of Serenity menu. Python is part of the ` Ports ` package and is actively being managed and maintained by [@linusg](https://github.com/linusg).
```

Comment:
I'm not sure this is very relevant to FontEditor

Reply:
I just let the first sentence remain. "Figure above shows where you can find FontEditor from inside the SerenityOS desktop.".